packages/apm/src/integrations/tracing.ts

```diff
@@ -436,6 +436,9 @@ export class Tracing implements Integration {
       ...transactionContext,
     }) as Transaction;
 
+    // We set the transaction here on the scope so error events pick up the trace context and attach it to the error
+    hub.configureScope(scope => scope.setSpan(Tracing._activeTransaction));
```

Comment:
This does it for the "idle transaction" -- what about manual instrumentation / `Sentry.startTransaction`?

cc @mitsuhiko

Reply:
Probably something to document.